backend/main.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const app = express()
+const GenericController = require('./GenericController')
+const port = 4000
+
+function get_models() {
+    const Sequelize = require("sequelize");
+    const initModels = require("./models/init-models").initModels;
+    var sequelize = new Sequelize("northwind", "root", "root", { dialect: 'mysql' });
```

Comment:
```var``` nu prea mai e folosit -- foloseste const sau let(atunci cand schimbi ceva in variabila). 

Sunt unele idei legate de hoisting etc, mai multe detalii aici: https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

Reply:
Am modificat codul. Daca este in regula, apasa _Resolve conversation_.